pkg/gpg/gpg_forwarding.go

```diff
@@ -88,6 +89,19 @@ func (g *GPGConf) ImportOwnerTrust() error {
 	return gpgOwnerTrustCmd.Run()
 }
 
+func (g *GPGConf) SetupGpgGitKey() error {
+	if g.GitKey != "" {
+		gitConfigCmd := exec.Command("git", []string{"config", "--global", "user.signingKey", g.GitKey}...)
+
+		out, err := gitConfigCmd.Output()
+		if err != nil {
+			return fmt.Errorf("git signkey: %s", string(out))
```

Comment:
nit: `errors.Wrap` for potential stacktrace?

Reply:
Done :+1: